champions/src/main/java/me/mykindos/betterpvp/champions/champions/builds/BuildManager.java

```diff
@@ -3,17 +3,44 @@
 import com.google.inject.Inject;
 import com.google.inject.Singleton;
 import lombok.Getter;
+import me.mykindos.betterpvp.champions.Champions;
+import me.mykindos.betterpvp.champions.champions.builds.event.LoadBuildsEvent;
 import me.mykindos.betterpvp.champions.champions.builds.repository.BuildRepository;
 import me.mykindos.betterpvp.core.framework.manager.Manager;
+import me.mykindos.betterpvp.core.utilities.UtilServer;
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Player;
 
 @Singleton
 @Getter
 public class BuildManager extends Manager<GamerBuilds> {
 
     private final BuildRepository buildRepository;
 
+    private final Champions champions;
+
     @Inject
-    public BuildManager(BuildRepository buildRepository) {
+    public BuildManager(BuildRepository buildRepository, Champions champions) {
         this.buildRepository = buildRepository;
+        this.champions = champions;
+    }
+
+    public void loadBuilds(Player player) {
+        GamerBuilds builds = new GamerBuilds(player.getUniqueId().toString());
+        getBuildRepository().loadBuilds(builds);
+        getBuildRepository().loadDefaultBuilds(builds);
+        addObject(player.getUniqueId().toString(), builds);
+        UtilServer.runTask(champions, () -> {
+            UtilServer.callEvent(new LoadBuildsEvent(player, builds));
+        });
     }
+
+    public void reloadBuilds() {
+        getObjects().clear();
+        for (Player player : Bukkit.getOnlinePlayers()) {
```

Comment:
What about players who are offline at the time of reload

Reply:
Whenever players logout, their data is removed. On reload, it just runs the same thing that happens for players on login, for every player. Players that have disabled skills in their builds will need to update their builds. If they do not, they will receive a message next time they login, as disabling a skill does not remove it from the database.

For #210 the issue was never offline players, as loading builds on login correctly did not include disabled skills (and did not take the points), with the exception of generating new default builds if a default skill was disabled.